Editor/Modules/TextureModule.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine.Profiling;
+using Unity.ProjectAuditor.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.Editor.Modules
+{
+    public enum TextureProperties
+    {
+        Name,
+        Format,
+        Shape,
+        ReadWrite,
+        StreamingMipMaps,
+        MinMipMapLevel,
+        Resolution,
+        SizeOnDisk,
+        Path,
+        Num
+    }
+
+    class TextureModule : ProjectAuditorModule
+    {
+        public static string[] searchTheseFolders;                 // Manually Choose Folders to do the FindAssets search within.
+
+
+        static readonly IssueLayout k_IssueLayout = new IssueLayout
+        {
+            category = IssueCategory.Texture,
+            properties = new[]
+            {
+                // new PropertyDefinition { type = PropertyType.Description, name = "Texture Analysis", longName = "Results of Texture Analysis "},
```

Comment:
I think we want a _PropertyType.Description_, see later comment

Reply:
Added in a PropertyType.Description with a descriptive Name and LongName. Is this what was intended?

Reply:
Did you push your changes?